src/transformers/modeling_bart.py

```diff
@@ -138,6 +138,10 @@ def dummy_inputs(self):
         }
         return dummy_inputs
 
+    @property
```

Comment:
would be happy to move this to `PretrainedModel`

Reply:
This and `next(self.parameters()).dtype` have roughly 20 appearance in the repo.

Reply:
yes, you can move this to `PretrainedModel` indeed